devenv.nix

```diff
@@ -145,11 +145,4 @@ in {
   env.CYPRESS_dbUser = lib.mkDefault "shopware";
   env.CYPRESS_dbPassword = lib.mkDefault "shopware";
   env.CYPRESS_dbName = lib.mkDefault "shopware";
-
-  scripts.build-updater.exec = ''
```

Comment:
not sure, if we need a replacement here :thinking:

Reply:
can be removed :)